components/tabbed-nav/src/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+// styled components
+const NavNavContainerli = styled('div')`
```

Comment:
I don't think you need this element? By default `div` has no `margin`, `padding` and has `display: block;` - I think this component would work if you removed this container completely toooooo